services/sendQuestionsAndAnswears.ts

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+import { EmailAnswer } from "../interfaces/email";
+import { QuestionWithAnswear } from "../interfaces/QuestionWithAnswear";
+
+export class sendQuestionsAndAnswears {
+  async send(
+    questionsWithAnswears: QuestionWithAnswear[]
+  ): Promise<EmailAnswer> {
+    const url = "http://localhost:3000/api/email/";
+    try {
+      console.log("tried to send q&a");
+      const response = await axios.post(url, { questionsWithAnswears });
+      const ConfirmationAnswer: EmailAnswer = response.data;
+      return ConfirmationAnswer;
+    } catch (err: any) {
+      throw new Error(
+        err.message + " Cannot connect to http://localhost:3000/api/email/"
```

Comment:
Instead of hardcoding the url, you could use the url variable declared above (on line 9)